handlers/slash.go

```diff
@@ -0,0 +1,29 @@
+package handlers
+
+import (
+	"log"
+
+	"github.com/bwmarrin/discordgo"
+)
+
+// https://github.com/bwmarrin/discordgo/blob/master/examples/slash_commands/main.go
+var (
+	Commands = []*discordgo.ApplicationCommand{
+		{
+			Name:        "ping",
+			Description: "Basic command, to check if the bot is running",
+		},
+	}
+
+	CommandHandlers = map[string]func(s *discordgo.Session, i *discordgo.InteractionCreate){
+		"ping": func(s *discordgo.Session, i *discordgo.InteractionCreate) {
```

Comment:
Ideally we move these to their own files for better separation, but we need to register these at some point, so this is probably okay for the time being. I'd file an issue to look for something cleaner.

Reply:
Maybe another issue for this as well?